views/components/table.html

```diff
@@ -0,0 +1,25 @@
+{% macro table(tableData) %}
+<article class="base--article" itemscope itemtype="http://schema.org/Table">
+  <header class="base--header">
+    <h1 class="base--h1" itemprop="name">{{tableData.name}}</h1>
+  </header>
+  <table class="base--table">
+    <thead class="base--thead">
+      <tr class="base--tr">
+        {% for header in tableData.data.headers %}
+        <th class="base--th">{{header}}</th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+        {% for row in tableData.data.row %}
```

Comment:
Please try https://github.com/punchcard-cms/punchcard/pull/555#discussion_r88254977 with the correct data model as described above.